### PR TITLE
[ADD] download_multiple_attachments_account_invoices

### PR DESCRIPTION
**Pull request by JorgeQuinteros**

### CODE REVIEW

File: download_multiple_attachments_account_invoices/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+###############################################################################
+#
+#    SDi Soluciones Digitales
```
Comment (JorgeQuinteros) [inline download_multiple_attachments_account_invoices/__manifest__.py]:
```suggestion
#    SDi Digital Group
```

File: download_multiple_attachments_account_invoices/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+{
+    'name': 'Download Multiple Attachments Account Invoices',
+    'version': '12.0.1.0.0',
+    'category': 'Tools',
+    'summary': '''This module allows to download attachments related to a
+     Account Invoices.''',
```
Comment (JorgeQuinteros) [inline download_multiple_attachments_account_invoices/__manifest__.py]:
```suggestion
         Account Invoices.''',
```

File: download_multiple_attachments_account_invoices/controller/main.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                file_name = invoice.date_invoice.strftime('%d_%m_%Y')
+                file_name += ' - ' + invoice.reference
+                if invoice.reference != attach_name:
+                    file_name += _('- Review - ') + attachment_id.name
+                file_path = attachment_id._full_path(file_store)
+                file_dict['%s:%s' % (file_store, file_name)] = dict(
+                    path=file_path,
+                    name=file_name)
+        zip_filename = datetime.now()
+        zip_filename = '%s.zip' % zip_filename
```
Comment (valentincastravete) [inline download_multiple_attachments_account_invoices/controller/main.py]:
```suggestion
        zip_filename = '%s.zip' % datetime.now()
```

**Review by valentincastravete — CHANGES_REQUESTED** [review]
Changes requested

**Review by valentincastravete — APPROVED** [review]

**Pull request merged**